.github/workflows/security-audit.yml

```diff
@@ -0,0 +1,19 @@
+name: Code Security Audit 
```

Comment:
This seems to possibly not even be required? From what I can gather this is primarily targeted towards Django.

I'd prefer changes which aren't required to not be made while I'm still being asked to review them.